src/main/java/org/mockito/verification/VerificationSucceededEvent.java

```diff
@@ -0,0 +1,51 @@
+package org.mockito.verification;
+
+import org.mockito.internal.verification.api.VerificationData;
```

Comment:
Let's avoid leaking internal classes and expose VerificationData as a public type (e.g. let's move it to public package)